### PR TITLE
Add a different cert chain to save

### PR DESCRIPTION
**Pull request by Wenxing-hou**
Fix: #836 

The bundle_requester.certchain1.der and bundle_requester.certchain.der have same leaf cert key. 
As same as bundle_responder.certchain1.der.

Signed-off-by: Wenxing Hou <wenxing.hou@intel.com>

### CODE REVIEW

File: unit_test/sample_key/readme.txt [hunk 1]
```diff
@@ -294,3 +294,23 @@ openssl asn1parse -in Shorter1024B_end_responder.cert -out Shorter1024B_end_resp
...
+==== More cert_chain to gen ====
+
+NOTE: The bundle_requester.certchain1.der and bundle_requester.certchain.der have same leaf cert key.
+As same as bundle_responder.certchain1.der.
+
+The script is:
+openssl req -nodes -x509 -days 3650 -newkey rsa:4096 -keyout ca1.key -out ca1.cert -sha512 -subj "//CN=intel test RSA CA"
+
+openssl x509 -req -in inter.req -out inter1.cert -CA ca1.cert -CAkey ca1.key -sha512 -days 3650 -set_serial 1 -extensions v3_inter -extfile ../openssl.cnf
+openssl x509 -req -in end_requester.req -out end_requester1.cert -CA inter1.cert -CAkey inter.key -sha512  -days 3650 -set_serial 2 -extensions v3_end -extfile ../openssl.cnf
```
Comment (jyao1) [inline unit_test/sample_key/readme.txt:308]: Would you please remove end_requester1.cert and end_responder1.cert ?

we just need ca1.cert and inter1.cert.

**Comment by jyao1** [comment]
please review what we have today, and check in everything.

**Comment by Wenxing-hou** [comment]
> please review what we have today, and check in everything.

I have added all generated file and updated readme.

**Review by jyao1 — APPROVED** [review]

**Comment by jyao1** [comment]
Merge it to unblock rest test.

**Pull request merged**